math/mcmcf90/files/0001-Makefile-restore-PPC-support.patch

```diff
@@ -0,0 +1,30 @@
+From 238fb1e1bcc9e491ce4220df66887ebbd64176ab Mon Sep 17 00:00:00 2001
```

Comment:
was this patch accepted upstream; please add a link to the PR/commit

Reply:
Here you go: https://github.com/mjlaine/mcmcf90/pull/1

If you look into the code, it is obvious this was there before, but someone decided to throw it away at some point. Moreover, PPC mk file is the original and Intel is likely a later addition (hence `maci` being used instead of `mac`).

Reply:
@reneeotten Since it got merged, I have dropped the patch and rebased to upstream.